src/module.c

```diff
@@ -13378,6 +13398,14 @@ int RM_RdbSave(RedisModuleCtx *ctx, RedisModuleRdbStream *stream, int flags) {
     return REDISMODULE_OK;
 }
 
+const char* RM_GetInternalSecret(RedisModuleCtx *ctx, size_t *len) {
+    UNUSED(ctx);
+    size_t local_len;
+    const char *secret = clusterGetInternalSecret(&local_len);
+    if (len) *len = local_len;
```

Comment:
I would say that the user must provide `len`, otherwise how would he know the size of the secret (we do not promise it is a C string right)?
If so maybe we should assert `len != NULL`

Reply:
Agree, I'll add an assertion